src/backend/marsha/settings.py

```diff
@@ -296,6 +299,7 @@ class Base(Configuration):
     JITSI_DOMAIN = values.Value("meet.jit.si")
     JITSI_CONFIG_OVERWRITE = values.DictValue({})
     JITSI_INTERFACE_CONFIG_OVERWRITE = values.DictValue({})
+    JITSI_SECRET_EXPIRATION = 3
```

Comment:
This is not directly related to jitsi. IMO you should not orefix it with `JITSI`. This variable name hould reflect the `live_pairing` feature. `LIVE_SECRET_EXPIRATION_MINUTES` ? `LIVE_SECRET_EXPIRATION_TIME` ? `LIVE_SECRET_EXPIRATION_DURATION` ?

Reply:
I renamed it to LIVE_PAIRING_EXPIRATION_MINUTES

Reply:
For coherence with other settings, we could count in seconds wdyt?